nginx/Dockerfile

```diff
@@ -10,22 +10,26 @@ RUN curl -Lo /tmp/consul_template_0.15.0_linux_amd64.zip https://releases.hashic
     unzip /tmp/consul_template_0.15.0_linux_amd64.zip && \
     mv consul-template /bin
 
-# Get ContainerPilot release
-ENV CONTAINERPILOT_VERSION 2.4.1
-RUN export CP_SHA1=198d96c8d7bfafb1ab6df96653c29701510b833c && \
-    curl -Lso /tmp/containerpilot.tar.gz "https://github.com/joyent/containerpilot/releases/download/${CONTAINERPILOT_VERSION}/containerpilot-${CONTAINERPILOT_VERSION}.tar.gz" && \
-    echo "${CP_SHA1}  /tmp/containerpilot.tar.gz" | sha1sum -c && \
-    tar zxf /tmp/containerpilot.tar.gz -C /bin && \
-    rm /tmp/containerpilot.tar.gz
+# Install ContainerPilot
+ENV CONTAINERPILOT_VER 3.0.0-RC1
+ENV CONTAINERPILOT /etc/containerpilot.json5
+
+RUN export CONTAINERPILOT_CHECKSUM=f67929d1c8567d31772085fc252338091a5f795c \
+    && curl -Lso /tmp/containerpilot.tar.gz \
+         "https://github.com/joyent/containerpilot/releases/download/${CONTAINERPILOT_VER}/containerpilot-${CONTAINERPILOT_VER}.tar.gz" \
+    && echo "${CONTAINERPILOT_CHECKSUM}  /tmp/containerpilot.tar.gz" | sha1sum -c \
+    && tar zxf /tmp/containerpilot.tar.gz -C /bin \
+    && rm /tmp/containerpilot.tar.gz
+
+# COPY ContainerPilot configuration
+COPY containerpilot.json5 /etc/containerpilot.json5
+ENV CONTAINERPILOT=/etc/containerpilot.json5
 
-COPY containerpilot.json /etc/containerpilot.json
-ENV CONTAINERPILOT=file:///etc/containerpilot.json
 COPY reload-nginx.sh /bin/
 COPY index.html /usr/share/nginx/html/
 COPY style.css /usr/share/nginx/html/
 COPY nginx.conf /etc/nginx/nginx.conf
 COPY nginx.conf.ctmpl /etc/containerpilot/
 
-
 EXPOSE 80
-CMD ["/bin/containerpilot", "nginx"]
+CMD ["/bin/containerpilot"]
```

Comment:
This is another case where the CMD is being truncated. Can you say more about the changed philosophy behind the change?